wfl/calculators/dft.py

```diff
@@ -61,26 +61,26 @@ def evaluate_dft(
 
     Returns
     -------
-    ConfigSet_in of configurations with calculated properties
+    ConfigSet of configurations with calculated properties
     """
     # defaults
     if dir_prefix is None:
         dir_prefix = f"run_{calculator_name}_"
 
     # choose the calculator
     if calculator_name == "CASTEP":
-        op = castep.evaluate_op
+        op = castep.evaluate_autopara_wrappable
     elif calculator_name == "VASP":
-        op = vasp.evaluate_op
+        op = vasp.evaluate_autopara_wrappable
     elif calculator_name == "QE":
-        op = espresso.evaluate_op
+        op = espresso.evaluate_autopara_wrappable
     else:
         raise ValueError(f"Calculator name `{calculator_name}` not understood")
 
     # run the calculation in parallel
-    return iterable_loop(
+    return autoparallelize(
         iterable=inputs,
-        configset_out=outputs,
+        outputspec=outputs,
         op=op,
         base_rundir=base_rundir,
```

Comment:
Do you think `wdir` might be a bit more self-explanatory/common than `rundir`? I think `base` bit could be slightly more self-explanatory too, would `root` be any better? So `wdir_root`?

Reply:
I don't find `wdir` to be any better - maybe `workdir`?

Reply:
Are you going to do that, or should I?

Reply:
I would prefer workdir

Reply:
> Are you going to do that, or should I?
I'll do that in a bit